backend/app/Http/Controllers/Auth/AuthenticatedSessionController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Auth\LoginRequest;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class AuthenticatedSessionController extends Controller
+{
+    /**
+     * Handle an incoming authentication request.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(LoginRequest $request)
+    {
+        $request->authenticate();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Call to an undefined method App\Http\Requests\Auth\LoginRequest::authenticate().